src/now.ts

```diff
@@ -92,15 +93,16 @@ export function perform<A>(comp: IO<A>): Now<Future<A>> {
 }
 
 class PerformIOStream<A> extends ActiveStream<A> {
+  node = new Node(this);
   constructor(s: Stream<IO<A>>) {
     super();
-    s.addListener(this);
+    s.addListener(this.node);
     this.state = State.Push;
   }
   push(io: IO<A>): void {
     runIO(io).then((a: A) => {
-      if (this.child !== undefined) {
-        this.child.push(a);
+      for (const child of this.children) {
+        child.push(a);
```

Comment:
Can we use `pushToChildren` here? `Stream` extends `Reactive` so streams should also have the `pushToChildren` method I think.